packages/jaeger-ui/src/utils/readJsonFile.test.js

```diff
@@ -72,4 +72,20 @@ describe('fileReader.readJsonFile', () => {
     const p = readJsonFile({ file });
     return expect(p).rejects.toMatchObject(expect.any(Error));
   });
+
+  it('loads JSON data with single resourceSpans array successfully', async () => {
+    const obj = { resourceSpans: [{ id: 1 }, { id: 2 }, { id: 3 }] };
```

Comment:
this is not a valid OTEL structure, it should be something like `resourceSpans > scopeSpans > spans`